CONTRIBUTING.md

```diff
@@ -21,39 +21,34 @@ Welcome to the MetricFlow developer community, we're thrilled to have you aboard
         - This is only required if you are developing with Postgres.
         - Follow the [instructions from Docker](https://docs.docker.com/get-docker/)
 3. [Create a fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of the [MetricFlow repo](https://github.com/dbt-labs/metricflow) and [clone it locally](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
-4. Activate a [Python virtual environment](https://docs.python.org/3/library/venv.html). While this is not required, it is *strongly* encouraged.
-    - We provide `make venv` and `make remove_venv` helpers for creating/deleting standard Python virtual envs. You may pass `VENV_NAME=your_custom_name` to override the default `venv` location.
-    - [conda](https://docs.conda.io/en/latest/) users may prefer [conda's environment management](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) instead.
-5. Install [Poetry](https://python-poetry.org/docs/) via `pip install poetry` - this is the tool we use to manage our build dependencies. Note, due to [an issue with poetry configurations using virtualenvs.create.false](https://github.com/python-poetry/poetry/issues/6459) some environments (e.g., Ubuntu) may experience dependency resolution problems with poetry 1.2.0, in which case you should make sure you have a virtual env set up prior to installation. If you really must use the global machine env you may be able to work around the problem by using `pip install poetry==1.1.15`.
-6. Run `make install` to get all of your dependencies loaded and ready for development
-    - This includes useful dev tools, including pre-commit for linting.
-    - You may run `pre-commit install` if you would like the linters to run prior to all local git commits
-7. OPTIONAL: install dbt dependencies. Developers working on dbt integrations will need to install these in order to work on those integrations and run the relevant tests. Any of the following commands should be sufficient for development purposes:
-    - `poetry install -E "dbt-postgres dbt-cloud"`
-    - `poetry install -E "dbt-redshift dbt-cloud"`
-    - `poetry install -E "dbt-snowflake dbt-cloud"`
-    - `poetry install -E "dbt-bigquery dbt-cloud"`
+4. Install [Hatch](https://github.com/pypa/hatch) via `make install-hatch` - this is the tool we use to manage our build
+dependencies.
 
 ## Start testing and development
 
-You're ready to start! Note all `make` and `poetry` commands should be run from your repository root unless otherwise indicated.
+You're ready to start! Note all `make` and `hatch` commands should be run from your repository root unless otherwise indicated.
+
+`pyproject.yaml` includes a definition for a Hatch environment named `dev_env` that is similar to a virtual environment
```

Comment:
Hmmm. How robust is this dev_env business?

Reply:
Depends on what you mean by robust?